### PR TITLE
Dog profile design

### PR DESCRIPTION
**Pull request by squashgray**
localhost_3000_create-dog (1)

### CODE REVIEW

File: src/pages/CreateDogProfilePage/CreateDogProfilePage.tsx [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+            )}
+            name="petAge"
+            control={control}
+            defaultValue=""
+          />
+          <Controller
+            render={({ field }) => (
+              <TextField
+                {...field}
+                className="materialUIInput"
```
Comment (allie-rae) [inline src/pages/CreateDogProfilePage/CreateDogProfilePage.tsx:124]: Is this className doing anything?

**Review by allie-rae — APPROVED** [review]
Amazing job!! This is my first ever review from a plane!! Buahahaha XD

File: src/pages/CreateDogProfilePage/CreateDogProfilePage.tsx [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+        <Stack spacing={4}>
+          <Typography variant="h1">Create Dog</Typography>
+          <Avatar sx={{ width: 56, height: 56, bgcolor: deepPurple[500] }}>
+            OP
+          </Avatar>
+          <Controller
+            render={({ field }) => (
+              <TextField
+                {...field}
+                className="materialUIInput"
```
Comment (allie-rae) [inline src/pages/CreateDogProfilePage/CreateDogProfilePage.tsx:87]:
```suggestion
```

File: src/pages/CreateDogProfilePage/CreateDogProfilePage.tsx [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+            )}
+            name="petName"
+            control={control}
+            defaultValue=""
+          />
+          <Controller
+            render={({ field }) => (
+              <TextField
+                {...field}
+                className="materialUIInput"
```
Comment (allie-rae) [inline src/pages/CreateDogProfilePage/CreateDogProfilePage.tsx:102]:
```suggestion
```

File: src/pages/CreateDogProfilePage/CreateDogProfilePage.tsx (hunk 1 above)
Comment (allie-rae) [inline src/pages/CreateDogProfilePage/CreateDogProfilePage.tsx:124]:
```suggestion
```

File: src/pages/CreateDogProfilePage/CreateDogProfilePage.tsx [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+              </RadioGroup>
+            )}
+            name="petSize"
+            control={control}
+          />
+          <Controller
+            render={({ field }) => (
+              <TextField
+                {...field}
+                className="materialUIInput"
```
Comment (allie-rae) [inline src/pages/CreateDogProfilePage/CreateDogProfilePage.tsx:264]:
```suggestion
```

File: src/pages/AboutPage/AboutPage.tsx [hunk 5]
```diff
@@ -80,6 +80,7 @@ const AboutPage: React.FC = (): React.ReactElement => {
       <Box sx={{ display: 'flex', flexWrap: 'wrap', mt: 4 }}>
         {badassDevs.map((dev) => {
           return (
+            // eslint-disable-next-line react/jsx-key
```
Comment (allie-rae) [inline src/pages/AboutPage/AboutPage.tsx:83]:
```suggestion
```

File: src/pages/AboutPage/AboutPage.tsx [hunk 6]
```diff
@@ -80,6 +80,7 @@ const AboutPage: React.FC = (): React.ReactElement => {
...
             <Card variant="outlined" sx={{ width: '240px', m: 1 }}>
```
Comment (allie-rae) [inline src/pages/AboutPage/AboutPage.tsx:84]:
```suggestion
            <Card variant="outlined" key={dev.name} sx={{ width: '240px', m: 1 }}>
```

**Review by zimashima — COMMENTED** [review]
``custom.ts`` and ``palette.ts`` should be their own new separate PR @allie-rae >:O

**Review by danimal-johnson — APPROVED** [review]
You guys will have to explain the reorganization of MaterialUI to me in our next standup. Otherwise this looks great (along with Allie's suggestions)

**Review by jae-finger — APPROVED** [review]
What a PR!

**Review by zimashima — APPROVED** [review]

**Pull request merged**